### ISSUE TITLE
MNasNet pre-trained mode accuracy mismatch?

### ISSUE THREAD

**Issue by tigert1998** [tensorflow/tpu#666]
I downloaded the `saved_model` format mnasnet model from https://storage.googleapis.com/mnasnet/checkpoints/mnasnet-a1.tar.gz. A text file in the folder claims the given `tflite` and `saved_model` model has the top1 accuracy of 75.180286% but I don't know the preprocessing method so I cannot reproduce it.

I have already tried the exact same preprocessing method in https://github.com/tensorflow/tpu/blob/master/models/official/mnasnet/preprocessing.py and tested it under ImageNet 2012 validation set but it only gave 73.534% top1 accuracy.

BTW, currently, https://github.com/tensorflow/tpu/blob/master/models/official/mnasnet/eval_ckpt_main.py seems useless because you don't even give a `ckpt` for researchers to test it.

**Comment by mgolub2** [comment]
I think 75% would be the test accuracy, so it's not that weird the validation set would be different? 

I'm pretty curious how the test accuracy was computed at all though, since the imagenet test server has been down for quite some time (> 1 year? ), at least for ILSVR2012 and ILSVR2017.
